fly.toml

```diff
@@ -9,6 +9,10 @@ kill_signal = "SIGINT"
 kill_timeout = "5s"
 swap_size_mb = 512
 
+[deploy]
+  # Purge the CDN cache after deploying so we pick up new or updated environment variables
+  release_command = "npx tsx app/utils/purge-cdn-cache.ts"
+
```

Comment:
This may be unnecessary and even undesirable... but I know I'll forget to manually purge after updating/adding env vars in the future.